xc/device.ml

```diff
@@ -1631,6 +1631,50 @@ let vgpu_args_of_nvidia domid vcpus (vgpu:Xenops_interface.Vgpu.nvidia) =
 	then ["--resume=" ^ resume_file]
 	else []
 
+let write_vgpu_data ~xs domid devid keys =
+	let path = xenops_vgpu_path domid devid in
+	xs.Xs.writev path keys
+
+let mxgpu_device_in_use ~xs physical_function =
+	(* Check if there is a /xenops/domain/<x>/device/vgpu/<y>/pf xenstore node
+	 * that is equal to the given physical_function. *)
+	let root = Device_common.xenops_domain_path in
+	try
+		xs.Xs.directory root
```

Comment:
It turns out we can cope with oversized packets, so it's fine (but should have a comment).